### ISSUE TITLE
Missing keyboard shortcuts in the file chooser (Macos X)

### ISSUE THREAD

**Issue by tlecomte** [Icy-imaging/Icy-Kernel#369]
Hi,

In the file chooser, with Macos X, the "enter" key is not associated with the "open" button. It's a little annoying... (I cannot do a selection with the keyboard only, whereas it's very desirable for example when you have to select the 200 individual files of a sequence with ctrl-a...)

Thanks for your efforts !

Timothée
